components/service-operator/apis/database/v1beta1/postgres_types.go

```diff
@@ -34,6 +34,7 @@ func init() {
 
 const (
 	Engine                       = "aurora-postgresql"
+	EngineVersion                = "10.13"
```

Comment:
question: have we confirm that all instances are currently provisioned with this version?

I'm assuming:

* if an instance exists with a different version, then this will be a problem
* we don't want to accidentally _downgrade_ an instance

Reply:
I think DCS is 10.7.

Reply:
May be better expose though the YAML instead?

Reply:
According to the AWS console, all the aurora postgres engines are 10.11, so this will cause an upgrade to happen to all of them. That's acceptable to me. I'll make sure a manual snapshot is taken of all of them before I promote the change to ensure we don't lose any data in the event of catastrophic failure. Given the usage of postgres in DCS, this won't be a user-impacting change.

Or: I could hard-code it to 10.11 rather than 10.13 and remove the fact an upgrade would happen.

I'd really like to avoid exposing it through the yaml.

Reply:
yep I think _upgrading_ is acceptable ... be nice to know if this causes downtime, I would think not for the HA instances